Frontend/DinoTracker/app/services/apollo-repository.ts

```diff
@@ -22,11 +23,24 @@ export default class ApolloRepository extends Service.extend({
     }
   }
 
-  public async createUser(variables: { name: string, username: string}): Promise<boolean>
+  public async createPaleontologist(variables: { name: string, username: string }): Promise<boolean>
   {
     try
     {
-      let response = await this.apollo.mutate({ mutation: createUserMutation, variables }, "createUser");
+      let response = await this.apollo.mutate({ mutation: createPaleontologistMutation, variables }, "createPaleontologist");
+      return response;
+    }
+    catch
+    {
+      return false;
+    }
+  }
+
+  public async createResearcher(variables: { name: string }): Promise<boolean>
+  {
+    try
```

Comment:
Don't worry about changing this here but I always like to let my exceptions bubble up as far as they can without going to the UI. Exceptions contain more information about what went wrong than just true or false so that lets me use that information in logic and build smarter UIs around it. For example, I can read the exception message and have my code do something special on certain exception messages. If you just use true or false though, you lose that capability.